crates/sdk/src/keygen/mod.rs

```diff
@@ -392,9 +393,8 @@ pub struct RootVerifierProvingKey {
     pub root_committed_exe: Arc<VmCommittedExe<RootSC>>,
     /// The constant trace heights, ordered by AIR ID.
     pub air_heights: Vec<usize>,
-    // The following is currently not used:
-    // The constant trace heights, ordered according to an internal ordering determined by the `NativeConfig`.
-    // pub internal_heights: VmComplexTraceHeights,
+    /// The constant trace heights in a semaantic way.
+    pub internal_heights: VmComplexTraceHeights,
```

Comment:
can we rename `internal_heights` to something else, because I got confused about this versus internal verifier?

Reply:
maybe just `override_heights` or `raw_heights`? It's just a permutation of `air_heights` in the chip complex ordering right?

Reply:
or just `heights` ...